data_pipeline/utils/data_store/bq_data_service.py

```diff
@@ -349,3 +351,70 @@ def load_new_data_in_temp_table_to_actual_table(
     query_job.result()  # Wait for the job to complete.
 
     LOGGER.info("Loaded table %s", table_id)
+
+
+def get_distinct_values_from_bq(
+            project_name: str,
+            dataset_name: str,
+            column_name: str,
+            table_name_source: str,
+            table_name_for_exclusion: str = None
+        ) -> pd.DataFrame:
+
+    sql = """
+        SELECT DISTINCT {{ column_name }}
+        FROM  `{{ project_name }}.{{ dataset_name }}.{{ table_name_source }}`
+        {% if table_name_for_exclusion %}
```

Comment:
comment: initially I thought you gone fancy with JinjaSql, but this seems a quite nice use-case for it